addons/hw_drivers/tools/helpers.py

```diff
@@ -341,6 +342,22 @@ def load_certificate():
         start_nginx_server()
     return True
 
+def delete_iot_handlers():
+    """
+    Delete all the drivers and interfaces
+    This is needed to avoid conflicts
+    with the newly downloaded drivers
+    """
+    try:
+        for directory in ['drivers', 'interfaces']:
+            path = file_path(f'hw_drivers/iot_handlers/{directory}')
+            iot_handlers = list_file_by_os(path)
+            for file in iot_handlers:
+                unlink_file(f"odoo/addons/hw_drivers/iot_handlers/{directory}/{file}")
```

Comment:
will it work on windows ? "/" is sometime not recognized as folder path (looks to be okay for `file_path` quikly looking at it's code)

Reply:
Yes it works, tested on both Windows and Rpi.
The `pathlib` translates and manages paths on its own independently of the OS, `'/'` being used as a default separator.
Proof from the documentation of pathlib:
![image](https://github.com/odoo/odoo/assets/36443074/5e544113-5b1f-4036-9778-3ebb85a74ec4)